src/execution/index.ts

```diff
@@ -15,4 +15,8 @@ export type {
 
 export { subscribe, createSourceEventStream } from './subscribe';
 
-export { getVariableValues, getDirectiveValues } from './values';
+export {
+  getVariableValues,
+  getArgumentValues,
```

Comment:
```suggestion
  getArgumentValues,
  getVariableValues,
```

@saihaj Please use consistent order in different `index.ts` files.
Changed to follow the same order as `src/index.ts`